src/avatar/avatar.component.spec.ts

```diff
@@ -14,11 +14,11 @@ describe("Avatar", () => {
                 InitAvatarComponent,
                 AvatarWithAttribsComponent,
                 IgxAvatarComponent,
-                IgxIconComponent,
                 InitIconAvatarComponent,
                 InitImageAvatarComponent,
                 InitAvatarWithAriaComponent
-            ]
+            ],
+            imports: [IgxIconModule.forRoot()]
```

Comment:
Same comment as in the previous PR. Document the usage of the `IgxIconService` as a singleton.